LibreNMS/OS/Panos.php

```diff
@@ -133,4 +138,17 @@ public function pollOS()
             $this->enableGraph('panos_activetunnels');
         }
     }
+
+    protected function memValid($storage)
+    {
+        if ($storage['hrStorageType'] == 'hrStorageOther') {
+            foreach ($this->validNetBufferMemory as $netBufferMemory) {
```

Comment:
Awesome, looking great.  Just one more tweak:
Str::contains()  accepts an array of needles, so we don't need  a loop.